policyengine_us/parameters/gov/states/ny/nysdps/con_edison_clean_heat/amount.yaml

```diff
@@ -0,0 +1,94 @@
+description: New York State Clean Heat rebate (conEdison)
+metadata:
+    unit: currency-USD #TODO: Includes both integer and strings. Need to be fixed.
+    breakdown:
+        - ny_clean_heat_family_type_category
+            - ny_clean_heat_heat_pump_category
+                - ny_clean_heat_dac_category
+                    - ny_clean_heat_home_category
+                        - ny_clean_heat_heat_pump_type_category
+                - ny_clean_heat_building_category
+                    - ny_clean_heat_description_category
+    period: year
+    label: New York State Clean Heat rebate (conEdison)
+    reference:
+        - title: New York State Clean Heat Con Edison Heat Pump Program Manual
+          href: https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf
+
+RESIDENTIAL:
+    ASHP:
+        NDAC:
+            SINGLE:
+                ASHPWIC:
+                    2024-03-01: 2_500
+                AWHPWDE:
+                    2024-03-01: 8_000
+            APARTMENT:
+                ASHPWDE:
+                    2024-03-01: 1_000
+                AWHPWDE:
+                    2024-03-01: 4_000
+        DAC:
+            SINGLE:
+                ASHPWIC:
+                    2024-03-01: 4_500
+                AWHPWDE:
+                    2024-03-01: 10_000
+            APARTMENT:
+                ASHPWDE:
+                    2024-03-01: 2_000
+                AWHPWDE:
+                    2024-03-01: 5_000
+    GSHP:
+        NDAC:
+            2024-03-01: 25_000
+        DAC:
+            2024-03-01: 35_000
+MULTIFAMILY:
```

Comment:
if residential and multifamily have different units and are computed differently lets separate those into 2 separate breakdown parameters